server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  attributes: {
+    id: {
+      type: 'string',
+      columnName: '_id'
+    },
+    createdAt: {
+      type: 'number',
+      autoCreatedAt: true,
+    },
+    updatedAt: {
+      type: 'number',
+      autoUpdatedAt: true,
+    },
+    fullName: {
+      type: 'string',
+      required: true
+    },
+    email: {
+      type: 'string',
+      required: true,
+      unique: true,
+      isEmail: true
+    },
+    campus: {
```

Comment:
if this is required, what is campus for someone in Focus lifestage going to be?

Reply:
thinking that focus lifestage is free to fill in their previous campus too, or people who have moved on to focus can still have that history of being part of campus. but I'm open to combining campus and lifestage or making these fields not required and they can be empty strings